articles/migrations/index.md

```diff
@@ -25,6 +25,102 @@ If you need help with the migration, create a ticket in our [Support Center](${e
+
+If you have any questions, create a ticket in our [Support Center](${env.DOMAIN_URL_SUPPORT}).
+
+### Introducing API Authorization with Third-Party Vendor APIs
+
+| Severity | Grace Period Start | Mandatory Opt-In|
+| --- | --- | --- |
+| Medium | Q1 2018 |  2018-06-01 |
+
+The mechanism by which you get tokens for third-party / vendor APIs (for example AWS, Firebase, and others) is being changed. It now works the same as any custom API, providing better consistency. This new architecture will be available in Q1 2018 and at that point the [/delegation](/api/authentication#delegation) endpoint will be deprecated. All clients must be updated prior to June 1, 2018. The migration guide will be available in Q1 2018.
+
+#### Am I affected by the change?
+
+If you are currently using [/delegation](/api/authentication#delegation) to provide third party authorization, your application will need to be updated.
+
+If you have any questions, create a ticket in our [Support Center](${env.DOMAIN_URL_SUPPORT}).
+
+### Deprecating the Usage of ID Tokens on the Auth0 Management API
+
+| Severity | Grace Period Start | Mandatory Opt-In|
+| --- | --- | --- |
+| Medium | 2017-12-21 |  2018-06-01 |
+
+We are deprecating the usage of [id tokens](/tokens/id-token) when calling [/users](/api/management/v2#!/Users/get_users_by_id) and [/device-credentials](/api/management/v2#!/Device_Credentials/get_device_credentials). We have moved to regular [id tokens](/tokens/access-token), and this is available now. Applications must be updated by June 1, 2018, when the ability to use id tokens will become unavailable. Migration guides will be available by February 2018.
+
+#### Am I affected by the change?
+
+If you are currently using [[id tokens](/tokens/id-token) to access any part of the Management API, your application will need to be updated.
+
+If you have any questions, create a ticket in our [Support Center](${env.DOMAIN_URL_SUPPORT}).
+
+### Improved OpenID Connect Interoperability in Auth0
+
+| Severity | Grace Period Start | Mandatory Opt-In|
+| --- | --- | --- |
+| Medium | 2017-12-21 |  2018-07-01 |
+
+The [userinfo](/api/authentication#get-user-info) endpoint is being updated to return [OIDC conformant user profile attributes](/user-profile/normalized/oidc). The most notable change is that `user_id` becomes `sub`. This will deprecate the [legacy Auth0 user profile](/user-profile/normalized/auth0) (in [userinfo](/api/authentication#get-user-info) and in [id tokens](/tokens/id-token)). Applications must be updated to use the new user profile before July 1, 2018. Migration guide will be available in late Q1 2018.
+
+#### Am I affected by the change?
+
+If you are currently using the [userinfo](/api/authentication#get-user-info) endpoint to retrieve user information, you are affected by this change and need to update your implementation.
```

Comment:
What if I am not using userinfo but use id tokens instead and get the user profile from there? Don't I have to update my application? Shouldn't this be more generic and say something like your application should expect the normalized user profile structure instead?